packages/node_modules/nav-frontend-stegindikator/src/stegindikator.tsx

```diff
@@ -0,0 +1,237 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+
+import StegindikatorSteg, { StegindikatorStegProps } from './stegindikator-steg';
+import { omit } from 'nav-frontend-js-utils';
+
+import 'nav-frontend-stegindikator-style';
+
+const cls = (state) => cn('stegindikator', {
+    'stegindikator--kompakt': state.kompakt
+});
+
+export interface StegindikatorProps {
+    steg: StegindikatorStegProps[];
+    visLabel: boolean;
+    kompakt: boolean;
+    aktivtSteg?: number;
+    onChange?: (index: number) => void;
+    onBeforeChange?: (index: number) => boolean;
+    autoResponsiv: boolean;
+}
+
+export interface StegindikatorState {
+    aktivtSteg: number;
+    visLabel: boolean;
+    kompakt: boolean;
+}
+
+class Stegindikator extends React.Component<StegindikatorProps, StegindikatorState> {
+
+    static Steg = StegindikatorSteg;
```

Comment:
Trenger vi å ha state i denne komponenten?
Evt. er det mulig å splitte ut state-håndtering i en egen komponent alà sånn som ekspanderbart-panel?

Reply:
Trenger state for `onChange` og `autoResponsiv`, men vi kan sikkert splitte det opp slik at vi også får en pure versjon hvis noen ønsker å håndtere alt dette manuelt.